### PR TITLE
explorer items views and feature button added

### PR DESCRIPTION
**Pull request by peratik**
## Changes
- styles updated
- queries updated
- views added
- moderator feature button added
- showAlert added

## Notion's card
https://www.notion.so/santiment/Explorer-post-views-70492ee03eff47cfb3bba24f39410038

## Checklist

- [x] I've performed a self-review, followed all rules from [Frontend style guide](https://www.notion.so/santiment/Front-end-style-guide-81750096b38c4bea9a29b14fd4ab8667)
- [x] If I make changes to another person's module, I've asked how to use it or request a review
- [x] I've updated the [documentation](https://github.com/santiment/academy), if necessary (Keyboard shortcuts, Account settings)
- [x] I've checked night mode, mobile & tablet screens (if have changes in UI)
- [x] I've added tests (if necessary)

## Screenshots or GIFs

image

image

### CODE REVIEW

**Review by DmitriVanGuard — APPROVED** [review]

**Comment by peratik** [comment]
We should wait until backend issue related to views get resolved.

File: src/pages/Explorer/Components/ActionButton.svelte [hunk 1]
```diff
@@ -9,13 +9,21 @@
   export let width = 16
   export let onClick = () => {}
   export let userVotes = 0
+  export let hasbackground = true
+  export let forceactive = false
```
Comment (DmitriVanGuard) [inline src/pages/Explorer/Components/ActionButton.svelte:13]: camelCase

File: src/pages/Explorer/Components/ActionButton.svelte [hunk 2]
```diff
@@ -9,13 +9,21 @@
...
+<Tooltip
+  dark
+  position="top"
+  align="center"
+  activeClass={hasbackground ? '$style.active' : '$style.passive'}
+  openDelay={200}
+>
   <div
-    class="actionbutton row hv-center"
+    class="actionbutton row hv-center {forceactive ? '$style.forceactive' : ''}"
```
Comment (DmitriVanGuard) [inline src/pages/Explorer/Components/ActionButton.svelte:26]: Use `class:[className]` [directive](https://svelte.dev/docs#template-syntax-element-directives-class-name)

**Review by DmitriVanGuard — CHANGES_REQUESTED** [review]
Use webkit's notifications store `import { notifications$ } from '@/ui/Notifications'`

File: src/pages/Explorer/Components/EditDialog.svelte [hunk 3]
```diff
@@ -32,9 +33,12 @@
...
       })
-      .catch((err) => {
-        // TODO: handle error and remove log
-        console.log(err.message)
+      .catch(() => {
+        notifications$.show({
+          variant: 'error',
+          title: 'Something went wrong',
+          description: 'Please try again or contact support',
+        })
```
Comment (DmitriVanGuard) [inline src/pages/Explorer/Components/EditDialog.svelte:41]: This error dispatcher is duplicated in multiple places (`src/pages/Explorer/Category/ExplorerCategory.svelte`, `src/pages/Explorer/Category/Widget.svelte`, etc). Move it to the function and reuse it

**Review by DmitriVanGuard — CHANGES_REQUESTED** [review]

**Review by DmitriVanGuard — APPROVED** [review]

**Pull request merged**